Makefile

```diff
@@ -9,16 +9,15 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 	targets/openshift/images.mk \
 )
 
-# Exclude e2e tests from unit testing
-GO_TEST_PACKAGES :=./pkg/... ./cmd/...
+IMAGE_REGISTRY :=registry.svc.ci.openshift.org
 
 # This will call a macro called "build-image" which will generate image specific targets based on the parameters:
 # $0 - macro name
-# $1 - target suffix
-# $2 - Dockerfile path
-# $3 - context directory for image build
-# It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./Dockerfile,.)
+# $1 - target name
+# $2 - image ref
+# $3 - Dockerfile path
+# $4 - context directory for image build
+$(call build-image,ocp-cluster-kube-scheduler-operator,$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-scheduler-operator, ./Dockerfile.rhel7,.)
```

Comment:
From https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/vendor/github.com/openshift/library-go/alpha-build-machinery/make/targets/openshift/images.mk#L28:

```sh
define build-image
$(eval $(call build-image-internal,$(1),$(2),$(3),$(4)))
endef
```

```sh
# $1 - target name
# $2 - image ref
# $3 - Dockerfile path
# $4 - context
define build-image-internal
```

Which, once rendered, will define Makefile's `image-ocp-cluster-kube-scheduler-operator` target:
```sh
$ make help | grep image
image-ocp-cluster-kube-scheduler-operator
images
```

Once `make images` is run, one will get:
```sh
$ imagebuilder --allow-pull -t registry.svc.ci.openshift.org/ocp/4.2:cluster-kube-scheduler-operator -f ./Dockerfile.rhel7 .
```

Looks good up to the point of running `imagebuilder` command.

Reply:
Nit: `$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-scheduler-operator` could be put under another variable, e.g.:

```
IMAGE_REGISTRY :=registry.svc.ci.openshift.org
TARGET_OPERATOR_IMAGE :=$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-scheduler-operator
...
$(call build-image,ocp-cluster-kube-scheduler-operator,$(TARGET_OPERATOR_IMAGE), ./Dockerfile.rhel7,.)
```

to improve readability.

Reply:
I think it less complicated without that extra variable. variables are intended to be set from outside - you can't change component name, just the registry.

let's take https://github.com/openshift/oc/blob/master/Makefile#L45 as a reference and keep it mostly similar